build.sbt

```diff
@@ -27,10 +25,6 @@ lazy val root = (project in file("."))
   .settings(scalaVersion := "2.13.8")
   .settings(headerSettings(A11yTest): _*)
   .settings(automateHeaderSettings(A11yTest))
```

Comment:
We can also remove `"-Wconf:src=src_managed/.*:s"` from the scalac options and `.*scalaxb.*` and `.*generated.*` from the scoverage excluded packages